### DISCUSSION TITLE
Donation possible?

### DISCUSSION THREAD

**Discussion by lukas-dne** [localsend/localsend#873] [Discussion (English, International)]
I know, maybe not the right place for this question, but anyway: is there a way to contribute financially to this project? 
Its so SO usefull and its so good, that you solved this stupid file-sharing issue between different os/devices. Thank you very much.

**Comment by Tienisto** [comment]
Hi, thanks for using LocalSend!

You can either donate through [Github](https://github.com/sponsors/Tienisto) or via [Ko-fi](https://ko-fi.com/tienisto)
